src/components/Modal/Modal.vue

```diff
@@ -295,7 +295,7 @@ export default {
 			type: String,
 			default: 'normal',
 			validator: size => {
-				return ['small', 'medium', 'normal', 'large', 'full'].indexOf(size) !== -1
+				return ['prompt', 'small', 'medium', 'normal', 'large', 'full'].indexOf(size) !== -1
```

Comment:
```suggestion
				return ['prompt', 'small', 'medium', 'normal', 'large', 'full'].includes(size)
```
Clearer semantics :)

Reply:
fixed